test/Faker/Provider/DateTimeTest.php

```diff
@@ -201,7 +201,7 @@ public function providerDateTimeBetween()
     {
         return [
             ['-1 year', false],
-            ['-1 year', null],
+            ['-1 year', ''],
```

Comment:
```
  1x: DateTime::__construct(): Passing null to parameter #1 ($datetime) of type string is deprecated
    1x in DateTimeTest::testDateTimeBetween from Faker\Test\Provider
```

Reply:
Thanks!

Reply:
The code on line 196 needs to be updated instead. We still need to test passing null on line 193.

Reply:
Fixed. Thanks!